images/custom-scorecard-tests/Dockerfile

```diff
@@ -0,0 +1,18 @@
+# Build the custom-scorecard-tests binary
+FROM --platform=x86_64 golang:1.17.9 as builder
+ARG TARGETARCH
+
+WORKDIR /workspace
+
+# Copy the go source
+COPY . .
+
+# Build
+RUN GOOS=linux GOARCH=$TARGETARCH BUILD_DIR=/workspace make GO_REQUIRED_MIN_VERSION:= build/custom-scorecard-tests
+
+# Final image.
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.5
```

Comment:
Is "latest" tag (https://catalog.redhat.com/software/containers/ubi8-minimal/5c64772edd19c77a158ea216) fine? If fine, how about using "latest"?

Reply:
not using this image anymore